src/components/ProductNameAndEdition.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { useEffect, useState } from "react";
+
+type Content = {
+  colours: string[];
+  image: string[];
+  index: number;
+  name: string;
+  edition?: string;
+  price: string;
+};
+
+export default function ProudctNameAndEdition({
+  content,
+}: {
+  content: Content;
+}) {
+  return (
+    <div className="relative flex flex-col items-center flex-wrap top-20 font-[440]">
```

Comment:
the 440 probably needs px at the end of it, good practice regardless to we know what metric it is

Reply:
updated.